pages/activity/[id]/agenda/[agendaId]/index.tsx

```diff
@@ -171,30 +177,58 @@ export default class AgendaDetailPage extends PureComponent<AgendaDetailPageProp
     return (
       <Container className="pt-5">
         <PageHead title={`${title} - ${name}`} />
-
-        <Row>
+        <Row className="my-3">
           <Col xs={12} sm={9}>
-            {this.renderHeader()}
-
-            <main className="my-2">{summary}</main>
+            <Row>
+              <Col xs={12} sm={9}>
+                {this.renderHeader()}
+
+                <main className="my-4">{summary + ''}</main>
+              </Col>
+              <Col xs={12} sm={3}>
+                <ActivityPeople
+                  names={mentors as string[]}
+                  avatars={(mentorAvatars as TableCellValue[]).map(file =>
+                    blobURLOf([file] as TableCellValue),
+                  )}
+                  positions={mentorPositions as string[]}
+                  summaries={mentorSummaries as string[]}
+                />
+                <h2>{t('attendee_ratings')}</h2>
+
+                <ScoreBar value={score} />
+              </Col>
+            </Row>
+
+            {fileInfo && <FileList data={fileInfo} />}
+            <div className="my-5">
+              <CommentBox
+                category="General"
+                categoryId="DIC_kwDOB88JLM4COLSV"
+              />
+            </div>
           </Col>
           <Col xs={12} sm={3}>
-            <ActivityPeople
-              names={mentors as string[]}
-              avatars={(mentorAvatars as TableCellValue[]).map(file =>
-                blobURLOf([file] as TableCellValue),
-              )}
-              positions={mentorPositions as string[]}
-              summaries={mentorSummaries as string[]}
-            />
-            <h2>观众评分</h2>
-
-            <ScoreBar value={score} />
+            {recommends[0] && (
+              <section id="related_agenda">
+                <h2 className="my-3">{t('related_agenda')}</h2>
+                <Row as="ol" className="list-unstyled g-4" xs={1}>
+                  {recommends
+                    .filter(agenda => agenda.title !== title)
+                    .map(agenda => (
+                      <Col as="li" key={agenda.id + ''}>
```

Comment:
```suggestion
                    .map(agenda => agenda.title !== title && (
                      <Col as="li" key={agenda.id + ''}>
```
JSX 会忽略假值，如上可减少遍历次数。